packages/react-native-editor/src/test/index.test.js

```diff
@@ -27,158 +29,202 @@ const initGutenberg = ( registerParams ) => {
-		expect( hookCallOrder ).toBeLessThan( onRenderEditorCallOrder );
-	} );
+			expect( blockList ).toHaveProperty( 'type', 'View' );
+			expect( console ).toHaveLogged( 'Hermes is: true' );
+		} );
 
-	it( 'dispatches "native.block_editor_props" hook before the editor is rendered', () => {
-		const applyFilters = jest.spyOn( wpHooks, 'applyFilters' );
-
-		// An empty component is provided in order to listen for render calls of the editor component.
-		const onRenderEditor = jest.fn();
-		const EditorComponent = () => {
-			onRenderEditor();
-			return null;
-		};
-		jest.mock( '../setup', () => ( {
-			__esModule: true,
-			default: jest.fn().mockReturnValue( <EditorComponent /> ),
-		} ) );
-
-		initGutenberg();
-
-		const hookCallIndex = 0;
-		// "invocationCallOrder" can be used to compare call orders between different mocks.
-		// Reference: https://git.io/JyBk0
-		const hookCallOrder =
-			applyFilters.mock.invocationCallOrder[ hookCallIndex ];
-		const onRenderEditorCallOrder =
-			onRenderEditor.mock.invocationCallOrder[ 0 ];
-		const hookName = applyFilters.mock.calls[ hookCallIndex ][ 0 ];
-
-		expect( hookName ).toBe( 'native.block_editor_props' );
-		expect( hookCallOrder ).toBeLessThan( onRenderEditorCallOrder );
-	} );
+		it( 'initializes the editor with initial HTML', async () => {
+			const { getByTestId } = initGutenberg(
+				{},
+				{ initialData: initialHtmlGutenberg }
+			);
 
-	it( 'dispatches "native.render" hook after the editor is rendered', () => {
-		const doAction = jest.spyOn( wpHooks, 'doAction' );
-
-		// An empty component is provided in order to listen for render calls of the editor component.
-		const onRenderEditor = jest.fn();
-		const EditorComponent = () => {
-			onRenderEditor();
-			return null;
-		};
-		jest.mock( '../setup', () => ( {
-			__esModule: true,
-			default: jest.fn().mockReturnValue( <EditorComponent /> ),
-		} ) );
-
-		initGutenberg();
-
-		const hookCallIndex = 1;
-		// "invocationCallOrder" can be used to compare call orders between different mocks.
-		// Reference: https://git.io/JyBk0
-		const hookCallOrder =
-			doAction.mock.invocationCallOrder[ hookCallIndex ];
-		const onRenderEditorCallOrder =
-			onRenderEditor.mock.invocationCallOrder[ 0 ];
-		const hookName = doAction.mock.calls[ hookCallIndex ][ 0 ];
-
-		expect( hookName ).toBe( 'native.render' );
-		expect( hookCallOrder ).toBeGreaterThan( onRenderEditorCallOrder );
-	} );
+			const blockList = await waitFor( () =>
+				getByTestId( 'block-list-wrapper' )
+			);
 
-	it( 'initializes the editor', () => {
-		const { getByTestId } = initGutenberg();
-		const blockList = waitFor( () => getByTestId( 'block-list-wrapper' ) );
-		expect( blockList ).toBeDefined();
+			expect( blockList ).toHaveProperty( 'type', 'View' );
+			expect( console ).toHaveLogged( 'Hermes is: true' );
+			// It's expected that some blocks are upgraded and inform about it (example: "Updated Block: core/cover")
+			expect( console ).toHaveInformed();
```

Comment:
This might be giving us a clue about using outdated versions of blocks in [`initialHtml` file](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/src/initial-html.js).

Reply:
By not passing arguments to `toHaveInformed`, I would imagine we expose ourselves to additional `console.info` calls accumulating without our knowledge. From reviewing the source of `@wordpress/jest-console`, I suppose there is no way to further constrain this assertion without passing the entirety of the explicit block update output, which is not manageable unfortunately. 



> This might be giving us a clue about using outdated versions of blocks in [`initialHtml` file](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/src/initial-html.js).

With this are you implying that we should fix the `initialHtml` rather than suppressing the `console.info` message here?

Reply:
> By not passing arguments to toHaveInformed, I would imagine we expose ourselves to additional console.info calls accumulating without our knowledge. From reviewing the source of @wordpress/jest-console, I suppose there is no way to further constrain this assertion without passing the entirety of the explicit block update output, which is not manageable unfortunately.

Exactly, there might be some cases where you could overlook extra or unexpected info logs using this function without passing arguments. In the case of this test, although we know the expected log output, I decided not to include it due to its size.

> With this are you implying that we should fix the initialHtml rather than suppressing the console.info message here?

Yep, I haven't investigated this further, but the fact that some blocks are being upgraded makes me wonder whether they should be reviewed, in case the HTML we're using for them is old.

Reply:
In any case, I think we could address this in a different PR, I'll open a ticket for this as a follow-up.